client/src/main/java/io/split/engine/common/SyncManagerImp.java

```diff
@@ -149,6 +165,14 @@ public void start() {
             } else {
                 startPollingMode();
             }
+
+            try {
+                _impressionManager.start();
+                _eventsTask.start();
+                _telemetrySyncTask.startScheduledTask();
+            } catch (Exception e) {
+                _log.error("Error trying to init synchronizer tasks.", e);
+            }
```

Comment:
Would it be a good idea to add a `try` block for each call in order to provide specific log messages for each task?

Reply:
I added a try for each start.